meinberlin/assets/scss/components/_poll.scss

```diff
@@ -296,10 +296,6 @@ $checkbox-size: 20px;
     padding: $spacer;
 }
 
-.poll-slider__answer {
-    padding-bottom: $spacer;
-}
-
```

Comment:
this is to stop longer answers from overlapping with the arrowes so should be retained or alternate solution added, currently they do overlap

Reply:
oops, i removed the wrong one, sorry! should have removed what you found in the other comment `..--highlight`